pkgs/data/fonts/lilex/default.nix

```diff
@@ -0,0 +1,34 @@
+{ lib, stdenvNoCC, fetchzip }:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "lilex";
+  version = "2.200";
+
+  src = fetchzip {
+    url = "https://github.com/mishamyrt/Lilex/releases/download/${version}/Lilex.zip";
+    sha256 = "sha256-MPQfqCMFMjcAlMos1o4bC+I+cvQYwr2TjI4Q03QeuaQ=";
+    stripRoot = false;
+  };
+
+
+  installPhase = ''
+    runHook preInstall
+
+    install -Dm644 variable/*.ttf -t $out/share/fonts/lilex
+
+    runHook postInstall
+  '';
+
+  meta = {
```

Comment:
Please keep the `meta` attributes sorted alphabetically and remove the empty line (line 29).